### PR TITLE
[Enhancement][RHEL6/7] add audit and display_login_attempts remediations

### LINKED ISSUE #691: Remediation for RHEL7 display_login_attempts

### LINKED ISSUE #701: Remediation for RHEL7 auditd_data_retention_num_logs

### LINKED ISSUE #702: Remediation for RHEL7 auditd_data_retention_max_log_file

### LINKED ISSUE #703: Remediation for RHEL7 auditd_data_retention_max_log_file_action

### LINKED ISSUE #704: Remediation for RHEL7 auditd_data_retention_action_mail_acct

### PR DESCRIPTION
**Pull request by redhatrises**
- Fixes #691
- Fixes #701
- Fixes #702
- Fixes #703
- Fixes #704

### CODE REVIEW

**Comment by isimluk** [comment]
This all looks good to me.

I am feeling good about merging this, but I haven't tested it. What do others think? Can we merge this?

**Comment by redhatrises** [comment]
@isimluk I am good with it.... but I create the PR so..... :)

File: shared/remediations/bash/display_login_attempts.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# platform = Red Hat Enterprise Linux 7
+
+if ! `grep -q showfailed /etc/pam.d/postlogin` ; then
```
Comment (shawndwells) [inline shared/remediations/bash/display_login_attempts.sh]: grep should be updated to ignore commented lines

Comment (shawndwells) [inline shared/remediations/bash/display_login_attempts.sh]: and likely to see if the line match begins with pam_succeed

**Comment by redhatrises** [comment]
@shawndwells updated the `display_login_attempts.sh` for you to review.

**Comment by redhatrises** [comment]
Bump for review.

**Pull request merged**